src/cuda_xengine.cu

```diff
@@ -130,6 +130,12 @@ __device__ static void operator+=( float4 &a, const float4 b ) {
  a = t;
 }
 
+#ifdef FIXED_POINT
+#define make_float4_rnd(x, y, z, w) make_float4(round(x), round(y), round(z), round(w))
```

Comment:
Well the correlator is presently outputting a float, so `round` is the way to go.  Unless you're suggesting that we should output an integer instead?